packages/theme-compatibility/webpack.config.js

```diff
@@ -1,31 +1,31 @@
 const path = require( 'path' );
 const FixStyleOnlyEntriesPlugin = require( 'webpack-fix-style-only-entries' );
+const SassConfig = require( '@automattic/calypso-build/webpack/sass' );
+const process = require( 'process' );
+
+const isDevelopment = process.env.NODE_ENV !== 'production';
 
 module.exports = {
 	mode: 'production',
 	entry: {
-		'leven': './stylesheets/leven.scss',
-		'leven-editor': './stylesheets/leven-editor.scss',
+		'crowdsignal': './src/themes/crowdsignal/base.scss',
```

Comment:
Nit: since this package is all about themes already, I think we can drop `/src/themes` and have just `/src/crowdsignal`, `/src/leven`, etc.

I guess the shared layer I mentioned would go into `/src/shared` as well then, the only difference being we don't build that one on its own.

Reply:
> Hmm... so right off the bat, I think I'd suggest to treat the themes a little differently:
> 
> Do you think it'd make sense if we split what currently makes up `crowdsignal/base` into two parts:
> 
> * A shared stylesheet, which would be a minimalistic layer that goes underneath every theme. The idea is this could be `@import`'ed directly inside other themes' stylesheets.
> * More specific stuff like colors etc. stays inside `crowdsignal.css` which now also imports that shared SCSS stylesheet.
> 
> If we do it that way, that's one less stylesheet we have to load when we boot our app or the editor. And `crowdsignal.css` can become a fully fledged theme, giving us the possibility to adapt or change it without necessarily affecting all other themes (which we still can do using the shared stylesheet).

Thanks for the comment @ice9js!
As we discussed, due to some dependencies order, we're not able to import a `shared` file inside the theme stylesheet and load a single file.
I created split the `crowdsignal` theme into two files: a `base` theme, which will be loaded along with the themes we support, and a `crowdsignal` theme itself.